__device-tests__/gutenberg-editor.test.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @format
+ * */
+
+import EditorPage from './pages/editor-page';
+import ParagraphBlock from './blocks/paragraph-block';
+import { rename, setupAppium, setupDriver } from './utils';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+
+describe( 'Gutenberg Editor tests', () => {
+	let appium;
+	let driver;
+	let editorPage;
+
+	const setupData = async () => {
+		await rename( 'src/app/initial-html.js', 'src/app/initial-html.tmp.js' );
+		await rename( 'src/app/initial-device-tests-html.js', 'src/app/initial-html.js' );
```

Comment:
Interesting trick here. I wonder if we can find a way to pass a parameter to Metro or the native (demo) app itself to bootstrap Gutenberg with some different initial data instead of having to replace the file in the filesystem.

I guess this can relate to the more general issue that the demo app needs to already be build outsite the Appium test run so, makes sense if we tackle this in a separate PR. Still, please share if you have any insight on this, thanks!

Reply:
> I guess this can relate to the more general issue that the demo app needs to already be build outsite the Appium test run so, makes sense if we tackle this in a separate PR. Still, please share if you have any insight on this, thanks!

Yeah might be best to tackle this in a separate PR, I tried approaches using babel-plugin-transform-inline-environment-variables and react-config, none seemed to work the way we need it to. For now, what I've done is move those steps out of the code and into the pre and post steps of the test, what do you think?